examples/model_predictive_controllers/pmsm_mpc_dq_current_control.ipynb

```diff
@@ -1110,9 +343,9 @@
    ],
    "source": [
     "%matplotlib notebook\n",
-    "\n",
+    "visu = MotorDashboard(state_plots=['i_sq', 'i_sd', 'u_sq', 'u_sd'], update_interval=10), # visualization\n",
```

Comment:
![image](https://user-images.githubusercontent.com/95769661/151168201-38949e8d-f2e3-459d-8222-c6c3637d0c27.png)
The comma in the end of the line make visu to a tuple in VSCode. Without comma in woks fine to initialize visu.

Reply:
Thanks, that was my bad.